README.md

```diff
@@ -53,15 +53,11 @@ Refer [concepts](doc/CONCEPTS.md) to understand the concepts referred to in this
 
 ## Usage
 
-Upgrade Guide to 3.x refer [here](UpgradeGuide.md)
```

Comment:
Upgrade guide should have guidance on adding ZIGGURAT_STREAM_ROUTER_<TOPIC_ENTITY>_UPGRADE_FROM config.

Reply:
yes, will add the upgrade guide in the wiki